gum/gumkernel.h

```diff
@@ -12,10 +12,14 @@
 G_BEGIN_DECLS
 
 GUM_API gboolean gum_kernel_api_is_available (void);
+GUM_API guint gum_kernel_query_page_size (void);
 GUM_API guint8 * gum_kernel_read (GumAddress address, gsize len,
     gsize * n_bytes_read);
 GUM_API gboolean gum_kernel_write (GumAddress address, const guint8 * bytes,
     gsize len);
+GUM_API gpointer gum_kernel_alloc_n_pages (guint n_pages);
+GUM_API gboolean gum_kernel_try_mprotect (gpointer address, gsize size,
+    GumPageProtection page_prot);
```

Comment:
Nit: To follow the ordering in e.g. `gumdukmemory.c` it would be good to move these two right before `read()` here and in the implementation, plus bindings.